antlr/src/main/antlr4/com/baeldung/antlr/ServerLog.g4

```diff
@@ -0,0 +1,34 @@
+grammar ServerLog;
+
+log : logEntry+ EOF ;
+logEntry : serverTime WS level WS message NEWLINE;
+serverTime : date WS dateTime;
+message : (WORD | WS)+ ;
+level : ('DEBUG' | 'INFO' | 'ERROR');
+date : NUMBER DASH month DASH NUMBER;
+dateTime : NUMBER COLON NUMBER COLON NUMBER;
+
+month
+   : 'Jan'
+   | 'Feb'
+   | 'Mar'
+   | 'Apr'
+   | 'May'
+   | 'Jun'
+   | 'Jul'
+   | 'Aug'
+   | 'Sep'
+   | 'Oct'
+   | 'Nov'
+   | 'Dec'
+   ;
+
+fragment LOWERCASE  : [a-z] ;
```

Comment:
Is the whitespacing issue here due to Github rendering it weird? Let's make sure what is in between the name and the colon are spaces so that everything lines up nicely.